packages/s3-credentials-endpoint/tests/test-index.js

```diff
@@ -119,11 +128,33 @@ test('An s3credential request with expired accessToken redirects to Oauth2 provi
     .set('Accept', 'application/json')
     .set('Cookie', [`accessToken=${accessTokenRecord.accessToken}`])
     .expect(307);
+  const authorizationUrl = buildEarthdataLoginClient().getAuthorizationUrl(response.req.path);
 
   t.is(response.status, 307);
   t.is(response.headers.location, authorizationUrl);
 });
 
+test('A redirect request returns a response with an unexpired cookie ', async (t) => {
+  const {
+    stubbedAccessToken,
+  } = context;
+
+  const response = await request(distributionApp)
+    .get('/redirect')
+    .query({ code: randomId('code'), state: 'original-path/s3credentials' })
```

Comment:
Nit: the `state` for this test isn't particularly import, but is there any reason it was prefixed with `original-path`?

Reply:
Not really